functests/5_latency_testing/latency_testing.go

```diff
@@ -206,28 +206,32 @@ func clearEnv() {
 
 func getValidValuesTests(toolToTest string) []latencyTest {
 	var testSet []latencyTest
-	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "5", testMaxLatency: guaranteedLatency, testCpus: "2", outputMsgs: []string{success}, toolToTest: toolToTest})
-	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "1", testMaxLatency: guaranteedLatency, testCpus: "5", outputMsgs: []string{success}, toolToTest: toolToTest})
-	testSet = append(testSet, latencyTest{testDelay: "1", testRun: "true", testRuntime: "2", testMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
+
+	//testRuntime: let runtime be 10 seconds for most of the tests and not less, that is to let the tools
+	//have their time to measure latency properly hence stabilizing the tests
+	//testCpus: for tests that expect a success output message, note that an even CPU number is needed, otherwise the test would fail with SMTAlignmentError
+	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "2", outputMsgs: []string{success}, toolToTest: toolToTest})
+	testSet = append(testSet, latencyTest{testDelay: "0", testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "6", outputMsgs: []string{success}, toolToTest: toolToTest})
```

Comment:
I get why testCpus was moved from 5 to 6, but why testRuntime was changed from 1 to 10?

Reply:
lsee line 210-211:
```
testRuntime: let runtime be 10 seconds for most of the tests and not less, that is to let the tools
have their time to measure latency properly hence stabilizing the tests
```
I have noticed that 5 is not good enough when we expect success message, that is, when expected latency is set to 1 and hwlatdetect runs for 5 seconds there are actually no samples to compare the latency with, thus adjusted the time to most of the tests to be 10 seconds

Reply:
ok, makes sense. Next time please considering splitting the commits such as
- one commit changes the cpu count to avoid SMTAlignmentError
- one commit change the test time to improve the stability of the tests

a commit should be a single unit of change and in general we should avoid mixing changes together unless they are really tightly coupled (this is not the case)

Reply:
you're right. I will take that into consideration, thank you for elaborating!